config.js

```diff
@@ -98,5 +98,17 @@ module.exports = {
       name: 'provider',
       url: 'https://provider.mainnet.oceanprotocol.com/spec'
     }
-  ]
+  ],
+  v3Versions: {
+    'ocean.js': 'v0.19.0',
```

Comment:
I think the latest ocean.js V3 version is: https://github.com/oceanprotocol/ocean.js/releases/tag/v0.20.2